id-authentication-default.properties

```diff
@@ -271,6 +271,8 @@ ida-topic-fraud-analysis=IDA_FRAUD_ANALYTICS
 ida-topic-auth-anonymous-profile=ANONYMOUS_PROFILE
 ida-topic-pmp-oidc-client-created=OIDC_CLIENT_CREATED
 ida-topic-pmp-oidc-client-updated=OIDC_CLIENT_UPDATED
+ida-topic-on-demand-template-extraction=AUTHENTICATION_ERRORS
+ida-topic-remove-id-status=REMOVE_ID_STATUS
```

Comment:
@mahammedtaheer - remove_id_status is part of ES-842 / array of handles release, is this also part of Java21 release?

Reply:
@kameshsr as you are working on java21 migration of IDA can you confirm if you need `ida-topic-remove-id-status=REMOVE_ID_STATUS` ??

Reply:
ida-topic-remove-id-status this property is not used in ida code of develop-java21 branch.

Reply:
Done